src/graphnet/models/utils.py

```diff
@@ -15,3 +16,18 @@ def calculate_xyzt_homophily(x, edge_index, batch):
     hz = homophily(edge_index, x[:,2], batch).reshape(-1,1)
     ht = homophily(edge_index, x[:,3], batch).reshape(-1,1)
     return hx, hy, hz, ht
+
+
+def calculate_distance_matrix(xyz_coords):
```

Comment:
```suggestion
def calculate_distance_matrix(xyz_coords: Tensor) -> Tensor:
```

Reply:
I think we should keep the name, but do what the name suggests: Return the matrix of pairwise, Euclidean distances. Currently, the matrix of squared distances is returned.